### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by opk12** [peterfajdiga/FastDraw#3]
Hello, I found this launcher on F-droid and tried it on Android 7.1. Version is 1.6. It crashes on launch.

```
FATAL EXCEPTION: main
Process: peterfajdiga.fastdraw, PID: 8582
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.graphics.drawable.Drawable.setCallback(android.graphics.drawable.Drawable$Callback)' on a null object reference
  at android.graphics.drawable.LayerDrawable.<init>(LayerDrawable.java:139)
  at android.graphics.drawable.LayerDrawable.<init>(LayerDrawable.java:117)
  at peterfajdiga.fastdraw.views.Drawables.createBgGradientDrawable(Drawables.java:81)
```

`LayerDrawable()` is passed a `new Drawable[2]` in [Drawables.java:81](https://github.com/peterfajdiga/FastDraw/blob/master/app/src/main/java/peterfajdiga/fastdraw/views/Drawables.java#L81). [The documentation](https://developer.android.com/reference/android/graphics/drawable/LayerDrawable#LayerDrawable(android.graphics.drawable.Drawable[])) does not tell if null is allowed inside the array. [The impl](https://cs.android.com/android/platform/superproject/main/+/main:frameworks/base/graphics/java/android/graphics/drawable/LayerDrawable.java;l=145?q=%2Fandroid%2Fgraphics%2Fdrawable%2FLayerDrawable&ss=android%2Fplatform%2Fsuperproject%2Fmain&hl=de) checks for `null` since [this commit](https://cs.android.com/android/_/android/platform/frameworks/base/+/abfcd0aa86f92afb46db2ffb9f5c4a45bab97a69?hl=de) from 2019.

**Comment by peterfajdiga** [comment]
Hi, thanks for the report! I've fixed the issue in branch issue_3 and [here](https://github.com/peterfajdiga/FastDraw/files/12907029/app-release.zip) is an apk with the fix.

Unfortunately, during fixing and testing this I've noticed that there's more differences in old Android versions than just that, the worst of which is that nested scrolling is completely broken in Android SDK versions under 28. The apk above should work mostly fine as long as you don't add a widget. But for the next releases, I'm limiting to SDK versions 28 and above.
